lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2169,13 +2169,13 @@ def _enable_ssh_on_windows(self, node: Node) -> None:
 
         log = node.log
         log.debug(
-            f"checking if SSH port {remote_node.public_port} is reachable "
+            f"checking if SSH port {remote_node._connection_info.port} is reachable "
```

Comment:
Thank you for making this fix! The _connection_info is private member, and shouldn't be used out of Node. Please use the `connection_info`. Because the wait_tcp_port_ready is always use both address and port of a Node, it's right time to change the interface to pass in the node only.

Reply:
@squirrelsc this PR got close after I reverted my changes and force pushed the changes. The new PR is here: https://github.com/microsoft/lisa/pull/2701